README.md

```diff
@@ -0,0 +1,14 @@
+<a href="https://zupimages.net/viewer.php?id=21/49/6rw0.png"><img src="https://zupimages.net/up/21/49/6rw0.png" alt="" /></a>
+
+## Description ✍🏻
+
+Website for EVeRetrofit company .
+Their project is to transform thermal vehicles into electric vehicles
+
+## Getting started 📌 ##
+
+1 - Fork the project
+
+2 - in your terminal install packages `npm install i`
+
```

Comment:
Don't forget to talk about esLint, prettier and the other configs because not everyone knows about them ;-)